lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+from . import config
+from .fixtures import TablesTest
+from .. import Integer
+from .. import testing
+from ..schema import Column
+from ..schema import Computed
+from ..schema import Table
+
+
+class ComputedReflectionFixtureTest(TablesTest):
```

Comment:
This is here just to avoid code duplication in `suite/test_reflection.py` and `engine/test_reflection.py`

maybe it can be removed or moved to fixture.py

Reply:
ok not a bad idea

Reply:
will move it once the test finish, so I'll also try to fix any issue that pops up

Reply:
will move it once the test finish, so I'll also try to fix any issue that pops up